Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/Annotation/AddIshAnnotation.cs

```diff
@@ -0,0 +1,345 @@
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "IshObjectGroup")]
+        [ValidateNotNullOrEmpty]
+        public string PubLogicalId { get; set; }
+
+        /// <summary>
+        /// <para type="description">Publication version</para>
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParametersGroup")]
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "IshObjectGroup")]
+        [ValidateNotNullOrEmpty]
+        public string PubVersion { get; set; }
+
+        /// <summary>
+        /// <para type="description">LogicalId of the content object</para>
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParametersGroup")]
+        [ValidateNotNullOrEmpty]
+        public string LogicalId { get; set; }
+
+        /// <summary>
+        /// <para type="description">Version of the content object</para>
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParametersGroup")]
+        [ValidateNotNullOrEmpty]
+        public string Version { get; set; }
+
+        /// <summary>
+        /// <para type="description">Language of the content object</para>
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParametersGroup")]
+        [ValidateNotNullOrEmpty]
+        public string Lng { get; set; }
+
+        /// <summary>
+        /// <para type="description">Type of the annotation</para>
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParametersGroup")]
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "IshObjectGroup")]
+        [ValidateNotNullOrEmpty]
+        public string Type { get; set; }
+
+        /// <summary>
+        /// <para type="description">Text of the annotation</para>
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParametersGroup")]
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "IshObjectGroup")]
+        [ValidateNotNullOrEmpty]
+        public string Text { get; set; }
+       
+        /// <summary>
+        /// <para type="description">Status of the annotation</para>
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParametersGroup")]
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = false, ParameterSetName = "IshObjectGroup")]
+        [ValidateNotNullOrEmpty]
+        public string Status { get; set; }
```

Comment:
Looking at the code I see that we are expecting the value of the status and not the element. Can you give an indicate for this by adding an example in the description? This also is the case for the Type, Category, Lng,